packages/client/src/services/api-etherscan.ts

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';

```

Comment:
we've been using native `fetch` in the codebase

Reply:
Let me remove axios and use native fetch.

Reply:
OOh. that's cool!